### PR TITLE
chore: default list methods to descending order

### PR DESCRIPTION
**Pull request by faroceann**
## Description
Default list methods to descending order.

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Review by alisherry — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,8 @@
-module github.com/workos/workos-go/v3
+module github.com/workos/workos-go/v4
 
 go 1.13
 
 require (
 	github.com/google/go-querystring v1.0.0
-	github.com/google/uuid v1.1.1
```
Comment (amygdalama) [inline go.mod:7]: what's the context for this change?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # WorkOS Go Library
```
Comment (amygdalama) [inline README.md:1]: are you planning to bump the version in separate PR? it's here https://github.com/workos/workos-go/blob/main/internal/workos/workos.go

File: go.mod (hunk 1 above)
Comment (faroceann) [reply go.mod:7]: I just ran `go mod tidy`, the dependency was previously removed in the codebase but not updated in the `go.mod`

File: README.md (hunk 2 above)
Comment (faroceann) [reply README.md:1]: Thanks, meant to bump this as well

**Review by amygdalama — APPROVED** [review]

**Pull request merged**